dev/start

```diff
@@ -15,7 +15,9 @@ Usage: start [options]
 
     --authn-iam     Starts with authn-iam/prod as authenticator
 
-    --authn-oidc    Starts with authn-oidc/okta as authenticator
+    --authn-oidc    Starts with authn-oidc/keycloak as authenticator
+
+    --oidc-adfs     Adds to authn-oidc adfs static env configuration
```

Comment:
What happens if we run `./start --oidc-adfs` without `--authn-oidc`? Is this the behaviour we want?

Reply:
Fix that :)